### ISSUE TITLE
Adding Servers to the application

### ISSUE THREAD

**Issue by NepXDNep** [jaketr00/Discord-Bot-Client#17]
Just an immediate-to-do list item would be adding servers to the application to further expand this.

**Comment by BlockyBoyRoll** [comment]
i agree, but this would also kinda cause chaos considering the power of bots

**Comment by AL1L** [comment]
I started to work on a new version, but haven't had the time to finish it
